openapi.yml

```diff
@@ -0,0 +1,381 @@
+openapi: '3.0.2'
+info:
+  title: SourceJump API
+  version: 1.0.0
+servers:
+  - url: https://sourcejump.net/api/v1
+paths:
+  /maps:
+    get:
+      summary: Return a list of all maps
+      tags: 
+        - maps
+      responses:
+        '200':
+          description: An array of Map objects
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/MapArray'
+        default:
+          description: Unexpected error
+  /maps/{map}:
+    get:
+      summary: Return detailed information about a specified map
+      tags: 
+        - maps
+      parameters:
+        - in: path
+          name: map
+          schema:
+            type: string
+          required: true
+          description: Name of the map
+      responses:
+        '200':
+          description: A Map opject
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Map'
+        '400':
+          $ref: '#/components/responses/Invalid'
+        '401':
+          $ref: '#/components/responses/Unauthorized'
```

Comment:
is this not gonna be public? prob dont need anything other than 200, 400 & 404

Reply:
lowkey copy/pasted the error responses across all paths, forgot to remove 401 from everything but /runs POST

Reply:
feels just like being employed again ✊